app/src/main/java/ch/epfl/sdp/LeaderboardAdapter.java

```diff
@@ -0,0 +1,58 @@
+package ch.epfl.sdp;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import java.util.List;
+
+public class LeaderboardAdapter extends RecyclerView.Adapter<LeaderboardAdapter.ViewHolder>{
+
+    private List<String> mUsernames;
+    private List<Double> mHealthPoints;
```

Comment:
As mentioned before, this should be list of Player

Reply:
Agree! but the reason is just as before!